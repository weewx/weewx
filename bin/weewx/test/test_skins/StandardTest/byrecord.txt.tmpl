#errorCatcher Echo
                   TEST ITERATING BY RECORDS
---------------------------------------------------------------------------------------
#for $_record in $day.records
$_record.dateTime $_record.outTemp
#end for
---------------------------------------------------------------------------------------
    
