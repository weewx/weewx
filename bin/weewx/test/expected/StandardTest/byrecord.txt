                   TEST ITERATING BY RECORDS
---------------------------------------------------------------------------------------
03-Sep-2010 00:10 63.3F
03-Sep-2010 00:20 62.5F
03-Sep-2010 00:30 61.6F
03-Sep-2010 00:40 60.7F
03-Sep-2010 00:50 59.9F
03-Sep-2010 01:00 59.0F
03-Sep-2010 01:10 58.1F
03-Sep-2010 01:20 57.2F
03-Sep-2010 01:30 56.4F
03-Sep-2010 01:40 55.5F
03-Sep-2010 01:50 54.6F
03-Sep-2010 02:00 53.8F
03-Sep-2010 02:10 53.0F
03-Sep-2010 02:20 52.1F
03-Sep-2010 02:30 51.3F
03-Sep-2010 02:40 50.5F
03-Sep-2010 02:50 49.7F
03-Sep-2010 03:00 48.9F
03-Sep-2010 03:10 48.2F
03-Sep-2010 03:20 47.5F
03-Sep-2010 03:30 46.8F
03-Sep-2010 03:40 46.1F
03-Sep-2010 03:50 45.4F
03-Sep-2010 04:00 44.8F
03-Sep-2010 04:10 44.2F
03-Sep-2010 04:20 43.6F
03-Sep-2010 04:30 43.0F
03-Sep-2010 04:40 42.5F
03-Sep-2010 04:50 42.0F
03-Sep-2010 05:00 41.6F
03-Sep-2010 05:10 41.2F
03-Sep-2010 05:20 40.8F
03-Sep-2010 05:30 40.4F
03-Sep-2010 05:40 40.1F
03-Sep-2010 05:50 39.8F
03-Sep-2010 06:00 39.6F
03-Sep-2010 06:10 39.3F
03-Sep-2010 06:20 39.2F
03-Sep-2010 06:30 39.0F
03-Sep-2010 06:40 38.9F
03-Sep-2010 06:50 38.9F
03-Sep-2010 07:00 38.8F
03-Sep-2010 07:10 38.9F
03-Sep-2010 07:20 38.9F
03-Sep-2010 07:30 39.0F
03-Sep-2010 07:40 39.1F
03-Sep-2010 07:50 39.3F
03-Sep-2010 08:00 39.5F
03-Sep-2010 08:10 39.7F
03-Sep-2010 08:20 40.0F
03-Sep-2010 08:30 40.3F
03-Sep-2010 08:40 40.7F
03-Sep-2010 08:50    N/A
03-Sep-2010 09:00 41.5F
03-Sep-2010 09:10 41.9F
03-Sep-2010 09:20 42.4F
03-Sep-2010 09:30 42.9F
03-Sep-2010 09:40 43.5F
03-Sep-2010 09:50 44.0F
03-Sep-2010 10:00 44.6F
03-Sep-2010 10:10 45.3F
03-Sep-2010 10:20 45.9F
03-Sep-2010 10:30 46.6F
03-Sep-2010 10:40 47.3F
03-Sep-2010 10:50 48.0F
03-Sep-2010 11:00 48.7F
03-Sep-2010 11:10 49.5F
03-Sep-2010 11:20 50.3F
---------------------------------------------------------------------------------------
    

